src/main/java/io/strimzi/kafka/bridge/http/HttpConfig.java

```diff
@@ -16,9 +16,9 @@
     private static final String HTTP_HOST = "HTTP_HOST";
```

Comment:
Do we still need these? Are these the env var definitions?

Reply:
As above, "http.host"

Reply:
These are now renamed (to priperties-like format) used as key for getting these properties.